tests/pages/author/design-questionnaire.page.js

```diff
@@ -1,34 +1,34 @@
 class DesignQuestionnaire {
   static setSectionTitle() {
-    return "#section-editor [aria-label='title']";
+    return "[data-testid='txt-section-title']";
   }
 
   static setQuestionTitle() {
-    return "#question-page-editor [aria-label='Question']";
+    return "[data-testid='txt-question-title']";
   }
 
   static setAnswerTitle() {
-    return "#question-page-editor [id='label']";
+    return "[data-test='txt-answer-label']";
```

Comment:
Why isn't this one `data-testid`?
It looks very inconsistent that we are using different attributes for elements on the same page

Reply:
The label is a simple text input, not a rich text editor. It's a pain that draft.js editor doesn't pass down the props the same as most other React components would but unfortunately that's a draft.js implementation detail. Throughout most of the app we're consistently using `data-test` attribute. This is the exception that breaks the rule.